projects/tinkerbell/charts/helm/sedfile.template

```diff
@@ -0,0 +1 @@
+s,0.4.4,${IMAGE_TAG},g
```

Comment:
i dont think this is your fault, but this stuff is super scary, seems easy to miss. make sure the updating instructions calls this out clearly.